client/my-sites/sidebar/sidebar.jsx

```diff
@@ -712,6 +711,9 @@ module.exports = React.createClass( {
 					</SidebarMenu>
 					: null
 				}
+				<SidebarFooter>
```

Comment:
I know "valid HTML" is mostly an intellectual thing at this point — but right now the rendered markup looks (simplified) like this: 

```
<ul class="sidebar">
	<div class="card current-site"> ... </div>
	<li class="sidebar__menu">
		<ul>
			<li class="stats"> ... </li>
			...
		</ul>
	</li>
	<div class="sidebar__footer"> ... </div>
</ul>
```

So there are a bunch of `div`s as direct descendants of an `ul`. I thought that wasn't kosher?